### ISSUE TITLE
Unexpected character encountered while parsing value: [. Path 'data.ca_chain', line 1, position 124.

### ISSUE THREAD

**Issue by Marusyk** [rajanadar/VaultSharp#73]
I'm using VaultSharp 0.11.0
```csharp
IAuthMethodInfo authMethod = new TokenAuthMethodInfo("secret");
var vaultClientSettings = new VaultClientSettings("http://localhost:8200", authMethod);
IVaultClient vaultClient = new VaultClient(vaultClientSettings);
var certificateCredentialsRequestOptions = new CertificateCredentialsRequestOptions();
certificateCredentialsRequestOptions.TimeToLive = "87600h";
certificateCredentialsRequestOptions.CommonName = "name1";
var certSecret = vaultClient.V1.Secrets.PKI.GetCredentialsAsync("rolename", certificateCredentialsRequestOptions, "pki_int").GetAwaiter().GetResult();
```

the last line throws:

> Newtonsoft.Json.JsonReaderException: Unexpected character encountered while parsing value: [. Path 'data.ca_chain', line 1, position 124.
>    at Newtonsoft.Json.JsonTextReader.ReadStringValue(ReadType readType)
>    at Newtonsoft.Json.JsonTextReader.ReadAsString()
>    at Newtonsoft.Json.JsonReader.ReadForType(JsonContract contract, Boolean hasConverter)
>    at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.PopulateObject(Object newObject, JsonReader reader, JsonObjectContract contract, JsonProperty member, String id)
>    at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateObject(JsonReader reader, Type objectType, JsonContract contract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerMember, Object existingValue)
>    at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.SetPropertyValue(JsonProperty property, JsonConverter propertyConverter, JsonContainerContract containerContract, JsonProperty containerProperty, JsonReader reader, Object target)
>    at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.PopulateObject(Object newObject, JsonReader reader, JsonObjectContract contract, JsonProperty member, String id)
>    at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateObject(JsonReader reader, Type objectType, JsonContract contract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerMember, Object existingValue)
>    at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.Deserialize(JsonReader reader, Type objectType, Boolean checkAdditionalContent)
>    at Newtonsoft.Json.JsonSerializer.DeserializeInternal(JsonReader reader, Type objectType)
>    at Newtonsoft.Json.JsonConvert.DeserializeObject(String value, Type type, JsonSerializerSettings settings)
>    at Newtonsoft.Json.JsonConvert.DeserializeObject[T](String value, JsonSerializerSettings settings)
>    at VaultSharp.Core.Polymath.MakeRequestAsync[TResponse](String resourcePath, HttpMethod httpMethod, Object requestData, IDictionary`2 headers, Boolean rawResponse, Action`1 postResponseAction)
>    at VaultSharp.Core.Polymath.MakeVaultApiRequest[TResponse](String resourcePath, HttpMethod httpMethod, Object requestData, Boolean rawResponse, Action`1 postResponseAction, String wrapTimeToLive, Boolean unauthenticated)
>    at VaultSharp.V1.SecretsEngines.PKI.PKISecretsEngineProvider.GetCredentialsAsync(String pkiRoleName, CertificateCredentialsRequestOptions certificateCredentialRequestOptions, String pkiBackendMountPoint, String wrapTimeToLive)
>    at SenderConsoleAppT1.Program.Main(String[] args) in C:\Users\roman.marusyk\source\repos\ConsoleApp1\ConsoleApp1\Program.cs:line 32

**Comment by brentdp** [comment]
I have experienced the same issue. I've made a pull request that should fix the issue, see #72  .

**Comment by Marusyk** [comment]
Fixed #72
